egs/librispeech/s5/README_chain_paper

```diff
@@ -0,0 +1,16 @@
+1. Command for generating nnet3 libirspeech tdnn+CE results:
+
+./local/nnet3/run_tdnn.sh
```

Comment:
Also please mention that we need to look at the decode_test_clean_fglarge results.